relay-metrics/src/aggregator.rs

```diff
@@ -693,7 +666,7 @@ impl MergeBuckets {
 
 /// Aggregator service interface.
 #[derive(Debug)]
-pub enum Aggregator {
+pub enum AggregatorManager {
```

Comment:
rename because now `Aggregator` refers to the data-part of the service that we extracted.